components/admin/requests/Header.tsx

```diff
@@ -107,28 +117,62 @@ export default function RequestHeader({
     return null;
   };
 
+  const displayShopifyUrl = paidThroughShopify && shopifyOrderID && shopifyOrderNumber;
+
+  const shopifyOrderUrl = displayShopifyUrl
+    ? `https://${process.env.NEXT_PUBLIC_SHOPIFY_DOMAIN}/admin/orders/${shopifyOrderID}`
+    : undefined;
+
   return (
     <Box textAlign="left">
-      <Link href="/admin" passHref>
+      <NextLink href="/admin" passHref>
         <Text textStyle="button-semibold" textColor="primary" as="a">
           <ChevronLeftIcon />
           All requests
         </Text>
-      </Link>
+      </NextLink>
       <Flex marginTop={5} alignItems="center">
         <Box>
           <Flex alignItems="center">
             <Text textStyle="display-large" as="h1" marginRight={3} textTransform="capitalize">
               {`${applicationType.toLowerCase()} Request`}
             </Text>
             {applicationStatus && <RequestStatusBadge variant={applicationStatus} />}
+            {paidThroughShopify && (
+              <Wrap>
+                <Badge variant={'ACTIVE'}>
+                  <Box pt="5px">
+                    <Image
+                      src="/assets/shopify-icon.svg"
+                      alt="Shopify Icon"
+                      height={70}
+                      width={70}
+                    ></Image>
+                  </Box>
+                </Badge>
+              </Wrap>
```

Comment:
could also make this a component if we think it'll be used in other places.

Reply:
Could we extract this to a component in `components/admin/requests/ShopifyBadge.tsx`? Could be used in the future